src/redux/constants.js

```diff
@@ -1,3 +1,3 @@
 export const NEWS_REQUESTED = 'NEWS_REQUESTED';
-export const NEWS_GOT = 'NEWS_GOT';
-export const NEWS_FAILED = 'NEWS_FAILED';
+export const NEWS_SUCCESS = 'NEWS_SUCCESS';
```

Comment:
AGAIN
Made new folder constants and remove this from redux folder, add this constants in object 'action types' and rename  constants like  GET_NEWS_SUCESS(ALL)